packages/react-native-eslint/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@noaignite/react-native-eslint",
+  "version": "1.0.0",
+  "description": "A comprehensive ESLint configuration for React Native projects with TypeScript support",
+  "main": "eslint.config.mjs",
+  "type": "module",
```

Comment:
todo: As we're using `"type": "module"` there should be no `.mjs` files. Setting `"type": "module"` makes esm the default, thus files should no longer be `.mjs` but `.js`.
TDLR: Rename `eslint.config.mjs` to `eslint.config.js` and re-point the `"main"` field.